### PR TITLE
data histogram

### PR DESCRIPTION
**Pull request by Borda**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you make sure to update the docs?   
- [x] Did you write any new necessary tests?  

## What does this PR do?

add labels histogram for use in Loss weights

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: kaggle_plantpatho/data.py [hunk 1]
```diff
@@ -117,17 +132,19 @@ def __len__(self) -> int:
...
+        lb = self.labels_lut['complex'] if torch.sum(lb) > 1 else torch.argmax(lb)
+        return int(lb)
+
+    def _prepare_labels(self) -> list:
+        labels = super()._prepare_labels()
+        return list(map(self._translate_labels, labels))
+
+    @property
+    def label_histogram(self) -> Tensor:
+        return torch.tensor(np.bincount(self.labels))
```
Comment (edgarriba) [inline kaggle_plantpatho/data.py:147]: there's also torch.bincount

File: kaggle_plantpatho/data.py [hunk 2]
```diff
@@ -15,6 +16,11 @@
 from torchvision import transforms as T
 from torchvision.transforms import InterpolationMode
 
+try:
+    from torchsampler import ImbalancedDatasetSampler
```
Comment (edgarriba) [inline kaggle_plantpatho/data.py:20]: why don't we enable by default?

**Review by edgarriba — APPROVED** [review]

File: kaggle_plantpatho/data.py (hunk 2 above)
Comment (Borda) [reply kaggle_plantpatho/data.py:20]: because it is not packaged on PyPI yet so if a user installs this package this sampler is missing
